components/authz-service/testhelpers/testhelpers.go

```diff
@@ -0,0 +1,213 @@
+package testhelpers
```

Comment:
I moved some functions we needed between testing package to a common package since go doesn't let you export functions defined in `_test.go` files outside of the module it resides in.